donjo-app/models/Keluarga_model.php

```diff
@@ -19,18 +16,7 @@ public function autocomplete($cari='')
 		if ($cari) $this->db->where("t.nama like '%$cari%'");
 		$data = $this->db->get()->result_array();
 
-		$str = autocomplete_data_ke_str($data);
-		return $str;
-	}
```

Comment:
Gunakan `$this->autocomplete_str($kolom, $tabel, $cari)`

Reply:
Ini yg diambil dri MY_Model pak @eddieridwan ??? Klw benar,  fuction tdk support untuk table yg di joint (hanya bisa digunakan jika tabel tunggal)